packages/config/lib/validation.ts

```diff
@@ -74,7 +75,7 @@ const isValidBrowser = (browser) => {
 /**
  * Validates the list of browsers.
  */
-const isValidBrowserList = (key, browsers) => {
+const isValidBrowserList = (key: string, browsers: any[]) => {
```

Comment:
I think this is `Browser` from `packages/types/src/browser.ts`?

Eg - if we are not going to type things quite strictly, I don't see a strong reason to change the file to `ts`.